packages/stunner-editors/kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/resources/org/kie/workbench/common/stunner/bpmn/resources/i18n/StunnerBPMNConstants.properties

```
@@ -177,8 +177,8 @@ org.kie.workbench.common.stunner.bpmn.definition.property.connectors.Priority.de
 org.kie.workbench.common.stunner.bpmn.definition.property.task.TaskPriority.label=Task Priority
 org.kie.workbench.common.stunner.bpmn.definition.property.task.TaskPriority.description=Sets the Priority of the Task
 org.kie.workbench.common.stunner.bpmn.definition.property.task.TaskPriority.helpMessage=Sets the priority of the Task.<br>\
-  It allows integer values, but also expressions. Expressions can be used in order to evaluate the integer value dynamically, instead of using some fixed number. <br>\
-  For instance, the priority can be set as just 25, or use the expression #{var} in order to set the value from evaluating the variable "var".
+ It allows integer values and expressions. Expressions can be used in order to evaluate the integer value dynamically, instead of using a fixed number.<br>\
```

You can avoid **<br>\** to break the line. 
```suggestion
You can use the expressions to evaluate the integer value dynamically, instead of using a fixed number. 

For an instance, you can set the priority to `25`, or the expression `#{var}` where the `var` variable contains a number.
```

---

Hello @kaldesai I was just updating the text and kept the original formatting, but I will commit the changes. Thank you!